resources/kcp/charts/kyma-environment-broker/templates/keb-db-job.yaml

```diff
@@ -87,55 +92,59 @@ spec:
               - name: APP_DATABASE_SSLROOTCERT
                 value: /secrets/cloudsql-sslrootcert/server-ca.pem
               - name: APP_BROKER_URL
-                value: "https://{{ .Values.host }}.{{ .Values.global.ingress.domainName }}"
+                value: "https://{{ $.Values.host }}.{{ $.Values.global.ingress.domainName }}"
               - name: APP_BROKER_TOKEN_URL
-                value: "https://oauth2.{{ .Values.global.ingress.domainName }}/oauth2/token"
+                value: "https://oauth2.{{ $.Values.global.ingress.domainName }}/oauth2/token"
               - name: APP_BROKER_CLIENT_ID
                 valueFrom:
                   secretKeyRef:
                     key: client_id
-                    name: {{ include "kyma-env-broker.fullname" . }}-oauth
+                    name: {{ include "kyma-env-broker.fullname" $ }}-oauth
               - name: APP_BROKER_CLIENT_SECRET
                 valueFrom:
                   secretKeyRef:
                     key: client_secret
-                    name: {{ include "kyma-env-broker.fullname" . }}-oauth
+                    name: {{ include "kyma-env-broker.fullname" $ }}-oauth
               - name: APP_BROKER_SCOPE
-                value: {{.Values.kebClient.scope}}
+                value: {{$.Values.kebClient.scope}}
             command:
-              - "./bin/environmentscleanup"
+              - "./bin/main"
```

Comment:
Why do we use inconsistently `./bin/main` and `/bin/main`. And the meaning is slightly different.